sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 4.3.0-beta.4 (Unreleased)
+## 4.3.0-beta.4 (2025-05-06)
 
 ### Features Added
 
-### Breaking Changes
+- Added support for `PreserveCertificateOrder` option.
 
-### Bugs Fixed
+### Breaking Changes
 
-### Other Changes
+- Updated API version to `7.6-preview.2`.

```

Comment:
Is this a breaking change? Did the API change? Would a code written previously not compile or start working diferently?